lib/Experian/IDAuth.pm

```diff
@@ -309,7 +309,15 @@ sub _get_result_proveid {
 
     return unless $credit_reference and $kyc_summary;
 
-    my $decision = {matches => []};
+    my $decision = {matches => [], kyc_summary_score => 0};
+
+    # calculate kyc summary score
+    
+    if ( ( my $nameCount = $kyc_summary->findvalue('FullNameAndAddress/Count') ) > 0
+    && ( my $dobCount = $kyc_summary->findvalue('DateOfBirth/Count') ) > 0 )
```

Comment:
The spacing and indentation seem a bit off from the usual style on this line. Is it possible to make it more consistent with elsewhere, while keeping `make tidy` happy? In addition, can we use "snake_case" rather than "camelCase" for variable names? I suggest 

    if ((my $name_count = $kyc_summary->findvalue('FullNameAndAddress/Count')) > 0 and
        (my $dob_count = $kyc_summary->findvalue('DateOfBirth/Count')) > 0) {
        ...

Reply:
In addition: Can these values ever be negative? If not, then it might be a good simplification to just drop the `> 0` condition test; zero is already false and non-zero is true.